changes/ce/fix-10462.en.md

```diff
@@ -0,0 +1,4 @@
+Deprecate config `broker.shared_dispatch_ack_enabled`.
```

Comment:
These changes should be part of  https://github.com/emqx/emqx/pull/10462. But why did it come to this PR 🤔